### ISSUE TITLE
Create SET

### ISSUE THREAD

**Issue by thiagonobrega** [dark-pid/hyperdrive#22]
Create a set that can set/add pid parameters as detailed in the following [documentation](https://github.com/dark-pid/hyperdrive/tree/feature/set_external_links/docs). The SET method receives the following parameters:

##### Parameters

> | name      |  type     | data type               | description                                                           |
> |----|---|---|---|
> | external_url     |  optional | str    | external url |
> | external_pid     |  optional | str    | external pid   |
> | payload          |  optional | json   | pid payload  |

The set method/route will use the functions already implemented (`set_payload`,`add_exeternal_pid`, and `add_url`).

In this version of the `/set` route (considering synchronous operation mode) **will only perform one of the three operations**  (`set_payload`,`add_exeternal_pid`, and `add_url`). 

In other words, if the request is sent with more than two variables (external_url, external_pid, or payload), the `set` will return an error :

> "Unable to execute multiple operations considering the Hyperdriver Synchronized Mode."
